.github/disclaimer.txt

```diff
@@ -1,2 +1,3 @@
-Copyright (C) 2014 Anaconda, Inc
+SPDX-FileCopyrightText: © 2012 Continuum Analytics, Inc. <http://continuum.io>
+SPDX-FileCopyrightText: © 2017 Anaconda, Inc. <https://www.anaconda.com>
```

Comment:
Let's not overcomplicate things and go with
```suggestion
SPDX-FileCopyrightText: © 2012 Anaconda, Inc. <https://www.anaconda.com>
```
The copyright remained with the same entity ([CIK: 0001538860](https://www.sec.gov/edgar/browse/?CIK=0001538860)), after all.
Let's just us the current name.

Reply:
```suggestion
SPDX-FileCopyrightText: © Anaconda, Inc. <https://www.anaconda.com>
```